src/test/java/org/openrewrite/java/migrate/UpgradeToJava17Test.java

```diff
@@ -527,4 +528,42 @@ void lombokBumpedGoingTo17() {
           )
         );
     }
+
+    @Test
+    void removedSSLSessionGetPeerCertificateChainMethodImplTest(){
```

Comment:
If possible, please split each recipe into its own test.

Reply:
Actually I thought unless we had multiple cases it was better to put them in the common Test.java

Reply:
I left the earlier comment here because I thought this test method was attempting to test more than one recipe. But that actually does not appear to be the case, so nothing needs to be changed here.